### ISSUE TITLE
remove use of args in component lifecycle hooks

### ISSUE THREAD

**Issue by jbailey4** [adopted-ember-addons/ember-collapsible-panel#35]
Deprecation details: https://emberjs.com/deprecations/v2.x/#toc_arguments-in-component-lifecycle-hooks

The `cp-panel` component is using the unsupported, now deprecated `attrs` arg in the `didReceiveAttrs` hook (https://github.com/tedconf/ember-collapsible-panel/blob/master/addon/components/cp-panel/component.js#L37). Instead component attrs should be retrieved via `Ember.get`.

**Comment by samselikoff** [comment]
#32 closes

**Comment by cah-brian-gantzler** [comment]
Thanks for doing this. Could we get a release that includes this? Hate to include master in my code.

**Comment by samselikoff** [comment]
@cah-briangantzler all set: https://github.com/tedconf/ember-collapsible-panel/releases/tag/v2.1.1

Btw in the future you don't have to use `master`, you can use specific SHAs in your package.json like this:

```js
"ember-collapsible-panel": "tedconf/ember-collapsible-panel#b6131ee7e84c05942cd80572fa6c9df37e037d7b"
```

**Comment by cah-brian-gantzler** [comment]
Thanks, not quite as nice as a specific version though, will give it a try

**Comment by samselikoff** [comment]
Most definitely, and I just released 2.1.1. Was just mentioning so that in the future you can use SHAs and other team members will get the same version when they `npm install`.
